src/frontend/src/service/api-service.ts

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+type Method = 'post' | 'delete' | 'get' | 'head' | 'delete' | 'options' | 'put';
+
+export type Request = {
+  url: string;
+  data?: object;
+  method: Method;
+};
+
+export default function apiService(request: Request) {
+  return axios
+    .request(request)
+    .then(response => response)
+    .catch(error => error);
```

Comment:
I'm not sure about two things here:

a. Why do we need this line?

        .then(response => response)

I don't think it does anything. You could delete it and the code would still function the same.

b. Why do we want this line?

    .catch(error => error)

If we catch errors here, anyone using `apiService` is still in the situation I described in https://github.com/codeforpdx/recordexpungPDX/pull/281#discussion_r307092169 except worse, because there's no obvious way to differentiate an error from a response.

Reply:
Ah, I see what you mean now.  I have removed the lines described in a and b, and updated the testing to work with the new code.  

I used npm i axios,  I'm not sure why my code was working without having axios in package.json.

Reply:
@SteelFace87 if the package is installed into your local `node_modules/` any code using that package will still work even if the package isn't listed in the `package.json`.